C/include/c4ReplicatorTypes.h

```diff
@@ -108,6 +109,7 @@ C4API_BEGIN_DECLS
     /** Information about a document that's been pushed or pulled. */
     typedef struct {
         C4HeapString collectionName;
+        C4HeapString scopeName;
```

Comment:
Should this be C4CollectionSpec for consistency?

Reply:
C4CollectionSpec is not on heap. I can only be passed around.

Reply:
OK

Reply:
So you did not mean, by using C4HeapString, to allow it to persist? I thought you wanted to keep the slice's buf after the caller deletes it? I can change it to C4CollectionSpec.